catalog/opensearch/catalog-opensearch-endpoint/src/main/java/org/codice/ddf/endpoints/OpenSearchEndpoint.java

```diff
@@ -360,32 +360,32 @@ private Response executeQuery(String format, OpenSearchQuery query, UriInfo ui,
                 }
             }
         } catch (UnsupportedQueryException ce) {
-            LOGGER.warn("Error executing query", ce);
-            response = Response.serverError()
-                    .entity(wrapStringInPreformattedTags(ce.getMessage()))
+            LOGGER.warn("Unsupported query", ce);
```

Comment:
Might be worth a method to remove duplicate code. Something like...
```java
private Response logAndThrow(String message, Exception e, Response resp) {
    LOGGER.warn(message, e);
    return resp.entity(wrapStringInPreformattedTags(message).build();
}
```
It doesn't save a ton since you still have to construct the `Response` object here but it's a little cleaner.